### PR TITLE
End-to-end encryption API enhancement

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 2.27.0  (January 2023)
```
Comment (jintgeorge) [inline CHANGELOG.md:1]: 2024?

**Review by jintgeorge — CHANGES_REQUESTED** [review]
Left a minor comment.

**Review by jintgeorge — APPROVED** [review]
LGTM

**Review by jvalli — APPROVED** [review]
LGTM!

**Pull request merged**